prov/sockets/src/sock_epoll.c

```diff
@@ -70,6 +70,7 @@ int sock_epoll_create(struct sock_epoll_set *set, int size)
 int sock_epoll_add(struct sock_epoll_set *set, int fd)
 {
 	struct epoll_event event;
+	memset(&event, 0, sizeof(struct epoll_event));
```

Comment:
Since we pass event in the following epoll_ctl call, valgrind finds it as uninitialized data